src/atomate2/forcefields/flows/elastic.py

```diff
@@ -45,6 +45,11 @@ class ElasticMaker(BaseElasticMaker):
     bulk_relax_maker : .ForceFieldRelaxMaker or None
         A maker to perform a tight relaxation on the bulk. Set to ``None`` to skip the
         bulk relaxation.
+    max_failed_deformations: int or float
```

Comment:
Doesn't show up below in the list of default parameters

Reply:
(Just to add: Typically, all parameters are repeated from the BaseMaker.)

Reply:
I can add it to the subclasses. But why redefining them, if the default value and definition are the same? I was actually confused by the fact that all the attributes are overwritten, while it is not strictly necessary.
This does not seem to be the case for all the Makers. For example the subclasses of `BaseVaspMaker` do not redefine the parameters of the base class.

Reply:
It was intitially intended to make the inheritance more transparent to users that directly read the code.

Reply:
I added it. I thought that it does not make sense to have some parameters overridden and some not, in this specific flow. We can remove it at a later stage if really needed.